contracts/src/L2/predeploys/L1GasPriceOracle.sol

```diff
@@ -48,6 +86,27 @@ contract L1GasPriceOracle is OwnableBase, IL1GasPriceOracle {
     /// @notice The address of whitelist contract.
     IWhitelist public whitelist;
 
+    /// @inheritdoc IL1GasPriceOracle
+    uint256 public override l1BlobBaseFee;
+
+    /// @inheritdoc IL1GasPriceOracle
+    uint256 public override commitScalar;
```

Comment:
What's the consideration for not reusing `scalar` as `commitScalar`? (Either way works, just curious.)

Reply:
I don't want to mix the meaning of `scalar` before and after Curie fork.